pkgs/applications/networking/p2p/deluge/default.nix

```diff
@@ -10,28 +20,58 @@ pythonPackages.buildPythonPackage rec {
     sha256 = "14d8kn2pvr1qv8mwqrxmj85jycr73vwfqz12hzag0ararbkfhyky";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/deluge-torrent/deluge/commit/d6c96d629183e8bab2167ef56457f994017e7c85.patch";
+      sha256 = "sha256-slGMt2bgp36pjDztJUXFeZNbzdJsus0s9ARRD6IpNUw=";
+      name = "fix_ngettext_warning.patch";
+    })
+  ] ++ lib.optional pythonPackages.isPy38 (fetchpatch {
```

Comment:
Does this break anything on other python versions? If not we shouldn't apply this conditionally to prevent bitrot.

Reply:
This patch is only needed for 3.8, but it will be part of the next deluge release, so we'll just drop the patches then.

Reply:
If by then the default python version migrated to 3.9 this patch will rot here. If it breaks nothing on other versions we can apply it unconditional to definitely notice when it breaks.

Reply:
Fair enough. Fixed.